cluster-autoscaler/utils/klogx/defaults.go

```diff
@@ -31,6 +31,9 @@ const (
 	// MaxNodesLoggedV5 is the maximum number of nodes for which we will
 	// log detailed information every loop at verbosity >= 5.
 	MaxNodesLoggedV5 = 1000
+	// MaxInstancesLogged is the maximum number of instances for which we will
+	// log detailed information for each Instance.List API call
+	MaxInstancesLogged = 20
```

Comment:
This is a GCE specific setting, I'd rather keep it under cloudprovider/gce and not let it spill into shared CA codebase

Reply:
Moved to cloudprovider/gce/autoscaling_gce_client.go